lld/ELF/Driver.cpp

```diff
@@ -1897,6 +1897,9 @@ static void setConfigs(Ctx &ctx, opt::InputArgList &args) {
       ErrAlways(ctx) << "cannot open --why-extract= file " << ctx.arg.whyExtract
                      << ": " << e.message();
   }
+
+  if (ctx.arg.osabi == ELFOSABI_OPENBSD)
```

Comment:
`ld.lld -z nobtcfi empty.o` has a false positive warning. Perhaps we need to move this variable back to readConfigs.